#! /bin/sh
patch -p0 -f $* < $0
exit $?

2009-06-20  Werner Koch  <wk@g10code.com>

	* gpastreamdecryptop.c (gpa_stream_decrypt_operation_new): Use
	"client-title" property.


--- src/gpastreamdecryptop.c	(revision 1008)
+++ src/gpastreamdecryptop.c	(working copy)
@@ -455,7 +455,7 @@
 		     "output_stream", output_stream,
                      "no-verify", no_verify,
                      "protocol", (int) protocol,
-                     "title", title,
+                     "client-title", title,
 		     NULL);
 
   return op;


